src/core/Akka/Actor/ActorRef.cs

```diff
@@ -121,43 +104,37 @@ public override IActorRefProvider Provider
         /// <param name="sender">TBD</param>
         protected override void TellInternal(object message, IActorRef sender)
         {
+            var handled = false;
```

Comment:
handled is a local variable, so if we attempt to send multiple messages to this `FutureActorRef<T>` this will evaluate to `true` each time, will it not?